include/llbuild/BuildSystem/BuildSystem.h

```diff
@@ -295,6 +295,22 @@ enum class QualityOfService {
 QualityOfService getDefaultQualityOfService();
 void setDefaultQualityOfService(QualityOfService level);
 
+// MARK: Execution Queue Scheduler Control
+
+enum class SchedulerAlgorithm {
+  /// Command name priority queue based scheduling [default]
+  commandNamePriority = 0,
+
+  /// First in, first out
+  fifo = 1
+};
```

Comment:
Unlike Swift, we have been using initial-uppercase for `enum class` constants... 🤷‍♂️

Reply:
Ah, I matched the style of QualityOfService right above.  😉

Probably worth cleaning up at some point, but I'll save it for later <rdar://problem/39958068>

Reply:
Well played